CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+```
+- [ ] What is language(s) for used in the example?
+- [ ] Is it focused on a particular module/software or an IOOS data service?
+- [ ] Can you provide a minimum example of the expected code and results in a notebook?
+
+Please provide a detailed description of the suggested example below:
+```
+
+### For the admin
+When the ticket gets created, make the following additions to the ticket:
+* Add the label [Notebook idea](https://github.com/ioos/ioos_code_lab/labels/Notebook%20idea).
+* Add the ticket to the [IOOS CodeLab](https://github.com/orgs/ioos/projects/1#card-49928448) project.
+* Put the card in the column [backlog](https://github.com/orgs/ioos/projects/1#column-5010196).
+
+## Building new notebooks
+If you would like to work on a notebook that has been proposed, follow these steps:
+1. [Create a fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of the [notebooks_demos](https://github.com/ioos/ioos_code_lab) repository.
+1. On your fork, [create a branch](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository) where you will start working on the new notebook. 
+   1. Give the branch a name that conveys what the new notebook is. For example, a notebook on converting data into Darwin Core could have a branch name of `data2dwc`. This makes it easier to know which branch is dedicated to which new notebook.
+1. On the new branch in your fork, start creating the new notebook. 
+   1. Notebooks are kept in the [jupyterbook/content/Code Gallery](https://github.com/ioos/ioos_code_lab/tree/master/jupyterbook/content/Code%20Gallery) directory of this repository.
+      1. There are three directories where notebooks can be placed:
+         1. `data_access_notebooks` - 
+         2. `data_analysis_and_visualization_notebooks` -
+         3. `data_management_notebooks` - 
+   2. Typically, it's best to copy an existing (working) notebook to a new file. If you are creating a python notebook, copy one of the python notebooks. If you are creating an R notebook, copy one of the R notebooks, etc
+   3. Name the notebook following the convention `DRAFT-[short name].ipynb`. Where, `[short name]` is an abbreviation for what the notebook will cover. Look at the existing notebook filenames to get a sense of the short names used. 
+1. Edit the notebook following the [Jupyter Notebook documentation](https://jupyter-notebook.readthedocs.io/en/stable/notebook.html).
+1. Once you feel comfortable with your notebook, [commit](https://github.com/git-guides/git-commit) your work to the branch on your fork on your local machine. 
+1. [Push](https://github.com/git-guides/git-push) your commit up to the branch on your fork in GitHub.
+1. [Create a Draft Pull Request (PR)](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).
```

Comment:
I would remove all the instructions for a draft PR. That can be confusing for newcomers we and want to lower the barrier. We, admins, can always change a PR to draft later if necessary.

Reply:
agreed.